core/variant/variant_op.h

```diff
@@ -1492,7 +1504,10 @@ class OperatorEvaluatorObjectHasPropertyString {
 	}
 	static inline void validated_evaluate(const Variant *left, const Variant *right, Variant *r_ret) {
 		Object *l = right->get_validated_object();
-		ERR_FAIL_NULL(l);
+		if (unlikely(!l)) {
+			*VariantGetInternalPtr<bool>::get_ptr(r_ret) = false;
+			ERR_FAIL_MSG("Invalid base object for 'in'");
```

Comment:
`ERR_FAIL` messages should by a full sentence ending with `.`:
```suggestion
			ERR_FAIL_MSG("Invalid base object for 'in'.");
```
(same below)